test/docs/promises.test.js

```diff
@@ -168,12 +172,12 @@ describe('promises docs', function () {
     }
     // acquit:ignore:end
     // Use bluebird
-    mongoose.Promise = require('bluebird');
+    global.Promise = require('bluebird');
```

Comment:
Remove this section entirely. We don't want to encourage setting `global.Promise`. Plus this will help us get rid of bluebird as a dep.